### DISCUSSION TITLE
How does disabling a Google Cloud account affect the Google Account?

### DISCUSSION THREAD

**Discussion by Enlight432** [immersive-translate/immersive-translate#1499] [Help] [answered]
### 概述 | Summary

![image](https://github.com/immersive-translate/immersive-translate/assets/123282471/a547ccd5-8b70-4a3c-921a-48d8619bfe4d)

 
Hello friends,If my Google account is disabled due to the above issue, will it cause any problems for my Google account !? tnx

**Comment by theowenyoung** [comment, marked as answer]
no, your google account is fine. and you can also contact the gemini support to restore your API

**Reply by Enlight432** [reply]
Thanks for the reply, will it solve the problem? I mean if I contact gemini support in the future will they reactivate my google cloud account?

**Reply by theowenyoung** [reply]
yes, it's ok.

**Reply by bcbcd666234** [reply]
My api is also blocked and the appeal Denied
